lib/Auth/Process/Tag2Attribute.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * COmanage authproc filter.
+ *
+ * This class is the authproc filter to get information about the user
+ * from the COmanage Registry database.
+ *
+ * Example configuration in the config/config.php
+ *
+ *    authproc.aa = [
+ *       ...
+ *       '74' => [
+ *            'class' => 'metaproc:Tag2Attribute',
+ *             'targetAttribute' => 'urn:oid:1.3.6.1.4.1.25178.1.2.10'
+ * 
+ */
+ 
+namespace SimpleSAML\Module\metaproc\Auth\Process;
+use SimpleSAML\Logger;
+use SimpleSAML\Metadata\MetaDataStorageHandler;
+
+class Tag2Attribute extends \SimpleSAML\Auth\ProcessingFilter
+{
+    private $targetAttribute = 'urn:oid:1.3.6.1.4.1.25178.1.2.10'; // schacHomeOrganizationType
```

Comment:
1. Change targetAttribute default to friendly name, i.e. `schacHomeOrganizationType`
2. Add '%replace' option (see https://github.com/simplesamlphp/simplesamlphp/blob/193db91883a0addab27c54506c64fb64e699984f/modules/core/lib/Auth/Process/AttributeAdd.php#L21)